google/cloud/bigquery/job/load.py

```diff
@@ -277,6 +277,27 @@ def null_marker(self):
     def null_marker(self, value):
         self._set_sub_prop("nullMarker", value)
 
+    @property
+    def projection_fields(self):
+        """Optional[List[str]]: If
```

Comment:
(nit) Let's annotate the types with type annotation syntax, we started doing that for all new code.

Reply:
Added! Also, TIL that pytype doesn't really work on macOS. I hit https://github.com/google/pytype/issues/661 when testing locally.